src/Compilers/CSharp/Portable/BoundTree/BoundNode.cs

```diff
@@ -18,24 +18,29 @@ internal abstract partial class BoundNode
         public readonly SyntaxNode Syntax;
 
         [Flags()]
-        private enum BoundNodeAttributes : byte
+        private enum BoundNodeAttributes : ushort
         {
             HasErrors = 1 << 0,
             CompilerGenerated = 1 << 1,
-            TopLevelNullableUnset = 0,
-            TopLevelNullable = 1 << 2,
-            TopLevelNonNullable = 1 << 3,
-            TopLevelUnknown = TopLevelNullable | TopLevelNonNullable,
-            TopLevelNullabilityMask = TopLevelUnknown,
+            IsSuppressed = 1 << 2,
+
+            // Bit 3: Whether nullability has been set
+            // Bit 4: 1 if the node is nullable, 0 if the node is not nullable
+            // Bits 5 and 6: 01 if the node is not annotated, 10 if the node is annotated, 00 if the node is disabled
+            TopLevelNullableInfoSet = 1 << 3,
```

Comment:
>TopLevelNullableInfoSet [](start = 12, length = 23)

I don't think we need this extra bit to know if nullability has been set. You can use the two later bits to encode that: 00 is unset, 01 is not-annotated, 10 is annotated, and 11 is disabled.